services/manifest-repo-export-service/pkg/cmd/server.go

```diff
@@ -151,6 +161,14 @@ func Run(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	var ddMetrics statsd.ClientInterface
+	if enableMetrics {
+		ddMetrics, err = statsd.New(DatatDogStatsAddr, statsd.WithNamespace("Kuberpult"))
+		if err != nil {
+			logger.FromContext(ctx).Fatal("datadog.metrics.error", zap.Error(err))
+		}
+		ctx = context.WithValue(ctx, repository.DdMetricsKey, ddMetrics)
```

Comment:
why do we need to change the context here?
I think this is not required
```suggestion
```

Reply:
I thought maybe in the future we may need the ddmetrics somewhere else for example in repository.go, exactly like what happens in cd-service.

Reply:
That may or may not happen. But we don't want to write code that is unused right now, so please remove it.

Reply:
Done!